src/TodoList.coffee

```diff
@@ -0,0 +1,53 @@
+define 'TodoList',
+[
+    './ViewInterface.coffee'
+],
+(ViewInterface) ->
+    class TodoList extends ViewInterface
+        items = ['First todo']
+        todoListSelector = document.querySelector('.todo-list')
+        init: ->
+            _render()
+            document.querySelector('.todo-list-form').addEventListener('submit', _addItem.bind(@))
```

Comment:
ocument.querySelector('.todo-list-form') - You use it twice at least. Move to variable